plugins/Morpheus/stylesheets/general/_forms.less

```diff
@@ -83,6 +82,11 @@ select {
             font-size: inherit;
         }
     }
+
+    .checkbox {
+      font-size: 14px;
+    }
```

Comment:
So this is how it's supposed to look?

![capture d ecran 2015-06-19 a 10 34 44](https://cloud.githubusercontent.com/assets/720328/8249855/d66c420a-166e-11e5-864d-e13d681e06d1.png)